kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateList.js

```diff
@@ -19,52 +20,70 @@ define([
         tr = t('tr'),
         th = t('th'),
         tbody = t('tbody'),
-        cssBaseClass = 'kb-job-state',
-        selectedJobCssClass = 'job-selected';
+        i = t('i'),
+        cssBaseClass = 'kb-job-status';
 
-
-    function renderTable() {
+    function createTable() {
         return table({
-            class: `table ${cssBaseClass}__table`
+            class: `${cssBaseClass}__table container`
```

Comment:
The idea with using BEM is to put all the styling information into one standalone css class, so that you can make changes to that class and be assured that it isn't going to affect anything else. You should be able to plonk a `kb-job-status__table` class in the middle of a chunk of html and be assured that it will be styled correctly. Can you copy over the styling attribs that you want from `container` into the `kb-job-status__table` scss (and similarly the table head class below)?

Reply:
Container is a bootstrap class for the column widths.